### PR TITLE
3 Secrets to Building Machine Learning Products

### PR DESCRIPTION
**Pull request by antonioalegria**
#3 Secrets to Building Amazing Machine Learning Products
- Speaker   : _Antonio Alegria_
- Available : _October 6th_ or _October 7th_
- Length    : _45 mins_
- Language  : _English_
## Description

**TL;DR**: _how to build and deploy real-time Machine Learning products that are magically scalable and automatically evolvable using off-the-shelf technologies_.

Data Science, Big Data, Bots, Complex Event Processing, Machine Learning, AI, Analytics, Deep Learning. Those are buzzwords that underscore a surge of mainstream products built upon the ability to use data to provide experiences that are, ironically, more human.

While data technologies have been exploding and becoming commoditized, using them effectively to build a product that delivers real value to users is still a black art. A lot of companies still use "gather data, think about it later" as a lazy excuse for a growth strategy but then fail to put that data to work.

I'm going to shine a light on the building blocks of Data Science and the Data Science lifecycle (from data to production to a continuously evolvable system). I will explore the fundamental recipe to build magical data products that leverage state of the art, readily available technology.
## Speaker Bio

I'm currently fighting crime as the Product Lead for Feedzai's Cloud-based, eCommerce Fraud Prevention service.

As a Product guy, I love designing and building systems that make sense of data, turning them into actionable insights for people and businesses in real-time and at scale. I am thrilled about solving challenging problems in elegant and efficient ways through the use of state-of-the art tools and by having an eye for quality.

In essence, I am a also a broadly-skilled Software Engineer that loves working on all sides of product development as an integrating force, thinking of new ways to farm insight from data and putting it into an easy to use, high-value product.

I have over 7 years experience in building real-time, scalable analytics and data processing systems in many domains, from Telecom to Payment Processing; from stealth network monitoring to real-time analytics; from Predictive Analytics to Fraud Prevention.
## Links
- **Blog:** http://antonioalegria.com (_deprecated_)
- **Company:** http://feedzai.com
- **Github:** http://github.com/antonioalegria

### CODE REVIEW

**Comment by antonioalegria** [comment]
3 Secrets to Building Machine Learning Products
===============================================

* Speaker   : *Antonio Alegria*
* Available : *October 6th* or *October 7th*
* Length    : *45 mins*
* Language  : *English*

Description
-----------

**TL;DR**: *how to build and deploy real-time Machine Learning products that are magically scalable and automatically evolvable using off-the-shelf technologies*.


Data Science, Big Data, Bots, Complex Event Processing, Machine Learning, AI, Analytics, Deep Learning. Those are buzzwords that underscore a surge of mainstream products built upon the ability to use data to provide experiences that are, ironically, more human.

While data technologies have been exploding and becoming commoditized, using them effectively to build a product that delivers real value to users is still a black art. A lot of companies still use "gather data, think about it later" as a lazy excuse for a growth strategy but then fail to put that data to work.

I'm going to shine a light on the building blocks of Data Science and the Data Science lifecycle (from data to production to a continuously evolvable system). I will explore the fundamental recipe to build magical data products that leverage state of the art, readily available technology.


Speaker Bio
-----------

I'm currently fighting crime as the Product Lead for Feedzai's Cloud-based, eCommerce Fraud Prevention service.

As a Product guy, I love designing and building systems that make sense of data, turning them into actionable insights for people and businesses in real-time and at scale. I am thrilled about solving challenging problems in elegant and efficient ways through the use of state-of-the art tools and by having an eye for quality.

In essence, I am a also a broadly-skilled Software Engineer that loves working on all sides of product development as an integrating force, thinking of new ways to farm insight from data and putting it into an easy to use, high-value product.

I have over 7 years experience in building real-time, scalable analytics and data processing systems in many domains, from Telecom to Payment Processing; from stealth network monitoring to real-time analytics; from Predictive Analytics to Fraud Prevention.


Links
-----

* **Blog:** http://antonioalegria.com (*deprecated*)
* **Company:** http://feedzai.com.com
* **Github:** http://github.com/antonioalegria

**Comment by celso** [comment]
Hi António, we need you to apply here so we can get your email etc to give you feedback on the talk review process: https://pixels.camp/apply/

Thanks!
